src/include/wiredtiger.in

```diff
@@ -49,12 +49,6 @@ extern "C" {
 #define	WT_ATTRIBUTE_LIBRARY_VISIBLE	__attribute__((visibility("default")))
 #endif
 
-#ifdef SWIG
```

Comment:
@ddanderson made that change in 8a1d2a7, I believe because the Python and Java wrapper code now includes `wiredtiger.h` explicitly. Don, is that correct?